WordPress/WordPressShareExtension/ShareViewController.swift

```diff
@@ -160,8 +162,60 @@ class ShareViewController: SLComposeServiceViewController {
     }
 }
 
+/// ShareViewController Extension: Encapsulates all of the class functions
+///
+private extension ShareViewController {
+    /// Removes all files from the Media upload directory.
+    ///
+    class func purgeUploadDirectory() {
```

Comment:
Better to move all this Media Folder related methods to a separate class and file?